changelog.d/6195.change

```diff
@@ -0,0 +1 @@
+Location sharing: Add labbs flag for live location sharing
```

Comment:
```suggestion
Location sharing: Add labs flag for live location sharing
```

Reply:
Also, another small thought. For the release notes it would be nice if there was a `feature` changelog to mention that live location sharing is now available for use.